File: dataset_reduzido/codenet/Java/Code/s063482767.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
	    // write your code here
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        PriorityQueue<Integer> queue = new PriorityQueue<>();
        for(int i = 0; i < n; i++) {
            queue.add(sc.nextInt());
        }
        int sum = 0;
        for(int i = 0; i < n - 1; i++) {
            sum += queue.poll();
        }

        // 出力
        System.out.println(queue.poll() < sum ? "Yes" : "No");
    }
}

File: output/Llama-2-100/codenet/Java/Python/s063482767.py
import sys

def main():
    # write your code here
    n = int(sys.stdin.readline())
    queue = PriorityQueue()
    for i in range(n):
        queue.add(int(sys.stdin.readline()))
    sum = 0
    for i in range(n - 1):
        sum += queue.get()

    # 出力
    if queue.get() < sum:
        print("Yes")
    else:
        print("No")

